packages/gradle/project.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "gradle",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/gradle/src",
+  "projectType": "library",
+  "targets": {
+    "nx-release-publish": {
+      "dependsOn": ["^nx-release-publish"],
+      "executor": "@nx/js:release-publish",
+      "options": {
+        "packageRoot": "build/packages/gradle"
+      }
+    },
```

Comment:
this is needed because Gradle is currently private. this target is only added for non-private repo, however, i still need to publish it for e2e tests:

in packages/nx/src/utils/package-json.ts:
```
  if (!packageJson.private && !res['nx-release-publish']) {
    res['nx-release-publish'] = {
      dependsOn: ['^nx-release-publish'],
      executor: '@nx/js:release-publish',
      options: {},
    };
  }
```